src/pages/scans/components/ScansModal.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import {
+    Bullseye,
+    Button,
+    EmptyState,
+    EmptyStateHeader,
+    EmptyStateIcon,
+    Modal,
+    ModalVariant,
+    Spinner,
+    Text,
+    TextContent,
+} from '@patternfly/react-core';
+import { ScanJobType, ScanType } from 'src/types';
+import { Table, Thead, Tr, Th, Tbody, Td, ThProps } from '@patternfly/react-table';
+import { useTranslation } from 'react-i18next';
+import moment from 'moment-timezone';
+import { DownloadIcon } from '@patternfly/react-icons';
+
+export interface ScansModalProps {
+    scan: ScanType;
+    scanJobs?: ScanJobType[];
+    onDownload: (number) => void;
+    onClose: () => void;
+}
+
+export const ScansModal: React.FC<ScansModalProps> = ({ scan, scanJobs, onDownload, onClose }) => {
+    const { t } = useTranslation();
+    const [activeSortIndex, setActiveSortIndex] = React.useState<number | undefined>();
+    const [activeSortDirection, setActiveSortDirection] = React.useState<'asc' | 'desc' | undefined>();
+    // const [selectedJobs, setSelectedJobs] = React.useState<number[]>([]);
+
+    const getSortParams = (columnIndex: number): ThProps['sort'] => ({
+        sortBy: {
+            index: activeSortIndex,
+            direction: activeSortDirection,
+            defaultDirection: 'asc'
+        },
+        onSort: (_event, index, direction) => {
+            setActiveSortIndex(index);
+            setActiveSortDirection(direction);
+        },
+        columnIndex
+    });
+
+    const formatDate = (date: Date) => {
+        return moment.utc(date).tz('America/New_York').format('DD MMMM Y, h:MM A z');
+    }
+    // const isJobSelected = (job: ScanJobType) => { return selectedJobs.includes(job.id); };
+    // const isJobSelectable = (job: ScanJobType) => { return true || job.status === "completed"; };
+    // const selectableJobs = scanJobs ? scanJobs.filter(isJobSelectable) : [];
+    // const onSelectJob = (job: ScanJobType, rowIndex: number, isSelecting: boolean) => {
+    //     setSelectedJobs((prevSelected) => {
+    //         const otherSelected = prevSelected.filter((j) => j !== job.id);
+    //         return isSelecting && isJobSelectable(job) ? [...otherSelected, job.id] : otherSelected;
+    //       });
+    // };
+    // const areAllJobsSelected = false;
+
+    // const selectAllJobs = (isSelecting: boolean = true) => {
+    //     setSelectedJobs(isSelecting ? selectableJobs.map((j) => j.id) : []);
+    // };
```

Comment:
Why do we have lots of commented-out code here and elsewhere in this function?

Reply:
from the original comment on this pr: "included some commented out checkbox progress for multi dl (for whenever that gets implemented in the rest api)"